examples/gl/pointsAsTexturesExample/bin/data/shaders_gles/shader.frag

```diff
@@ -1,12 +1,14 @@
-#ifdef GL_ES
-// define default precision for float, vec, mat.
-precision highp float;
-#endif
+#version 300 es
+// fragment shader
+
+precision lowp float;
+
+in vec4 in_Color;
+
+out vec4 out_Color;
 
 uniform sampler2D tex;
 
 void main (void) {
-    
-    gl_FragColor = texture2D(tex, gl_TexCoord[0].st) * gl_Color;
-    
-}
\ No newline at end of file
+	out_Color = vec4(in_Color.rgb, texture(tex, gl_PointCoord).a);
```

Comment:
shouldn't this be: 
`out_Color = texture(tex, gl_PointCoord) * in_Color;`

the code above is basically just using the point sprite texture's alpha

Reply:
Probably, yes. I am happy if you or anyone can improve it. I am not really experienced with shaders.

Reply:
Cool - yeah both work for me on desktop. 
If this works for Emscripten could you update the PR and we'll merge it? 

`out_Color = texture(tex, gl_PointCoord) * in_Color;`

Otherwise let me know and we'll leave it as is. 

Thanks!

Reply:
Thanks. That works with Emscripten too and looks better. Edited it.